system/jlib/jstats.cpp

```diff
@@ -2439,7 +2440,8 @@ void CRuntimeStatisticCollection::set(const CRuntimeStatisticCollection & other,
     {
         StatisticKind kind = other.getKind(i);
         unsigned __int64 value = other.getStatisticValue(kind);
-        setStatistic(kind, value, node);
+        if (value)
```

Comment:
not sure about this check, why it's needed, If all the stat kind's are now correctly in the target.
Wondered if it might cause problems by not setting if 0 in some situations.